network-api/networkapi/campaign/views.py

```diff
@@ -11,6 +11,8 @@
 
 from networkapi.wagtailpages.models import Petition
 
+gs_sqs = False
```

Comment:
changes to this file are mostly because I needed to do localhost testing and these changes are critical for that. I can file this as separate PR if desired, though.